### ISSUE TITLE
Does ES work on Banana Pi?

### ISSUE THREAD

**Issue by aldoblack** [Aloshi/EmulationStation#354]
I was wondering if ES works on Banana Pi with Raspbian installed?

**Comment by Aloshi** [comment]
I have no idea, since I don't have a Banana Pi, but I would guess it works fine.

**Comment by felix3008** [comment]
Yes, it works very nice! (I am currently working on a banana pi arcade cabinet).
It is a lot of work tho. You need to compile sdl2 without opengl support (important) and then build ES. 
I have made a simple script that compiles sdl2 on archlinux somewhere on the 6th page of [this forum thread](http://forum.lemaker.org/thread-6212-1-1-1.html).

You will probably want to build retroarch and a few libretro cores as well.
